diff --git a/src/Controller/SummerCampController.php b/src/Controller/SummerCampController.php
index f59c03d..3271da4 100644
--- a/src/Controller/SummerCampController.php
+++ b/src/Controller/SummerCampController.php
@@ -2,6 +2,7 @@
 
 namespace App\Controller;
 
+use Psr\Log\LoggerInterface;
 use Sensio\Bundle\FrameworkExtraBundle\Configuration\Route;
 use Symfony\Bundle\FrameworkBundle\Controller\Controller;
 
@@ -17,8 +18,10 @@ class SummerCampController extends Controller
     /**
      * @Route("/speakers")
      */
-    public function speakers()
+    public function speakers(LoggerInterface $logger)
     {
+        $logger->info('We are logging stuff!!!!');
+
         return $this->render('summer_camp/speakers.twig', [
             'speakers' => ['Marco', 'Paula', 'Antonio', 'Marcello']
         ]);
